client.js

```diff
@@ -111,16 +94,20 @@ $(function() {
         '</h1></a><li>';
     }
     $('.result-list').html(html);
-    $('.result-link').click(function() {
-      ga(
```

Comment:
Argument per line function calls are confusing and shouldn't be used as long as the full function call can be fit in a single line with a width of 80-100 characters.

Reply:
Makes a lot of sense to me, we need to have sanitization practices.